t/57-oracle-datetime.t

```diff
@@ -83,6 +89,73 @@ isa-ok $sth.column-types[0], DateTime, 'CURRENT_TIMESTAMP is DateTime';
 $sth.execute;
 my $datetime2 = $sth.row[0];
 isnt $datetime, $datetime2,		    'Server drift';
+
 diag $datetime2.Instant - $datetime.Instant;
 
-$dbh.execute($dropper);
+ok $dbh.execute('DELETE FROM test_datetime WHERE adate IS NOT NULL'), 'Delete rows';
+ok $dbh.dispose, 'Close session';
+
+note '#+ -------------------------------------------------------- +';
+note '## Issue #204 - ADDED Missing TIMESTAMP Support (BEGIN)';
+note '#+ -------------------------------------------------------- +';
+
+%con-parms = :database<XE>, :username<TESTUSER>, :password<Testpass>, :!AutoCommit;
+
+try {
+  $dbh = DBIish.connect('Oracle', |%con-parms);
+  CATCH {
+	    when X::DBIish::LibraryMissing | X::DBDish::ConnectionFailed {
+		diag "$_\nCan't continue.";
+	    }
+            default { .rethrow; }
+  }
+}
+
+# session mgmt comming soon
+is $dbh.do('ALTER SESSION SET time_zone               = \'-00:00\''), 0, 'ALTER SESSION ...';
+is $dbh.do('ALTER SESSION SET nls_date_format         = \'YYYY-MM-DD"T"HH24:MI:SS"Z"\''), 0, 'ALTER SESSION ...';
+is $dbh.do('ALTER SESSION SET nls_timestamp_format    = \'YYYY-MM-DD"T"HH24:MI:SS"Z"\''), 0, 'ALTER SESSION ...';
+is $dbh.do('ALTER SESSION SET nls_timestamp_tz_format = \'YYYY-MM-DD"T"HH24:MI:SS"Z"\''), 0, 'ALTER SESSION ...';
+is $dbh.execute('SELECT * FROM test_datetime').allrows, (), 'Perfect table is empty!';
+
+$sth = $dbh.prepare(
+    q|INSERT INTO test_datetime
+    FIELDS (adate, atimestamp0, atimestamp6, atimestamp0tz, atimestamp6tz, atimestamp0ltz, atimestamp6ltz)
+    VALUES (?,?,?,?,?,?,?)|);
+$now = DateTime.now.truncated-to('second').in-timezone(0);
+
+say '# Tasty timestamp: ', $now;
+lives-ok {
+    $sth.execute(
+        $now, # Need an ISO full date
+	$now, $now, $now, $now, $now, $now
+    );
+},                                           'Can insert Perl6 values';
```

Comment:
This should read Raku not Perl6.

Reply:
I agree; but I didn't write the comment; I.E. you'll find the same elsewhere in the test; I followed the example; since I'd already gotten grief for too many changes

Reply:
Line 133 is displayed as added by your commit in github.

Reply:
I now says Raku and the comment was nuked